backend/instagram_backend/apps/users/views.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import generics
+from models import User, Follower
+from serializers import UserSerializer, FollowerSerializer
+
+
+class UserListView(generics.ListAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+    def get_queryset(self):
+        return User.objects.values('username', 'profile_picture')
```

Comment:
why are you getting values out of this queryset? If you want only `username` and `profile_picture` in your response, you should do this on serializer level, not here